src/shared/services/configuration/map-markers.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable global-require */
+import L from 'leaflet';
+
+export const smallMarkerIcon = global.window.L.icon({
+  iconUrl: 'https://map.data.amsterdam.nl/dist/images/svg/marker.svg',
+  iconSize: [20, 20],
+  iconAnchor: [10, 19],
+});
+
+export const markerIcon = L.icon({
+  iconUrl: 'https://map.data.amsterdam.nl/dist/images/svg/marker.svg',
+  iconSize: [40, 40],
+  iconAnchor: [20, 39],
+});
```

Comment:
Isn't one icon enough? Why would we need a big and a small icon?

Reply:
small icon is used in the incident overview.

Reply:
Hm, can't we use a static image for that? We don't need an interactive map component in the incident preview mode.

Reply:
we could but is more work than this while the component already does what it should do. The map has to be rendered anyway in the preview.